crates/nu-engine/Cargo.toml

```diff
@@ -13,8 +13,8 @@ nu-path = { path = "../nu-path", version = "0.70.1"  }
 nu-glob = { path = "../nu-glob", version = "0.70.1" }
 nu-utils = { path = "../nu-utils", version = "0.70.1"  }
 
-chrono = { version="0.4.21", features=["serde"] }
-sysinfo = "0.26.2"
+chrono = { version="0.4.21", features = ["std"], default-features = false }
```

Comment:
if we take this out, can we still send dates to a plugin?

Reply:
Oh, or is that we only need that feature on some crates and not others?

Reply:
some crates does not need those features, the benefit may be we improve where in the build graph we may require it.